src/System.Private.ServiceModel/src/System/ServiceModel/Channels/RTSocketConnection.cs

```diff
@@ -393,7 +393,20 @@ protected override void WriteCore(byte[] buffer, int offset, int size, bool imme
                 }
                 throw;
             }
+        }
 
+        protected override void TraceWriteStart(int size, bool async)
+        {
```

Comment:
Is this an exact copy of the code for CoreCLRSocketConnection? Not for this PR, but we will eventually need to rename this file to follow the new naming guidelines and probably make it a partial type to avoid duplicating code.